admin/class-parcelpro-shipping.php

```diff
@@ -231,7 +256,14 @@ public function calculate_shipping($package = array())
                                     $maatwerk_type = '_' . $rule[ 'type-id' ] ;
                                 }
 
-                                $this->prepare_service($key . $maatwerk_type, $rule[ 'method-title' ], $rule[ 'price' ], $this->custom_services[ $key ][ 'order' ], key_exists('type-id', $rule) ? $rule[ 'type-id' ] : null, isset($rule[ 'servicepunt' ])  ? $rule[ 'servicepunt' ] : null);
+                                $this->prepare_service(
+                                    $key . $maatwerk_type,
+                                    $rule[ 'method-title' ] . ' ' . $formattedDeliveryDate,
```

Comment:
Adding the space here instead of before the opening paren in the `$formattedDeliveryDate` feels a bit weird. No big deal though, so also fine to leave it, but did want to mention it.